coord/src/rmr/test/test_cluster.c

```diff
@@ -14,11 +14,26 @@
 
 void testEndpoint() {
 
-  MREndpoint ep;
-  mu_assert_int_eq(REDIS_OK, MREndpoint_Parse("localhost:6379", &ep));
+  MREndpoint ep;
 
+  mu_assert_int_eq(REDIS_OK, MREndpoint_Parse("localhost:6379", &ep));
   mu_check(!strcmp(ep.host, "localhost"));
-  mu_assert_int_eq(6379, ep.port);
+  mu_assert_int_eq(6379, ep.port);
+
+  // ipv6 tests
+  mu_assert_int_eq(REDIS_OK, MREndpoint_Parse("::0:6379", &ep));
+  mu_check(!strcmp(ep.host, "::0"));
+  mu_assert_int_eq(6379, ep.port);
+
+  mu_assert_int_eq(REDIS_OK, MREndpoint_Parse("[fe80::8749:8fe8:f206:2ab9]:6379", &ep));

```

Comment:
Cover also brackets without port?

Reply:
Maybe use a different port than the default?

Reply:
Added